lib/Redmine/Api/AbstractApi.php

```diff
@@ -117,6 +117,36 @@ protected function sanitizeParams(array $defaults, array $params)
             array($this, 'isNotNull')
         );
     }
+
+    /**
+     * Generate a Redmine compatible URL-encoded query string
+     * 
+     * @param mixed $data
+     * @param string $parentKey
+     * @param bool $isNested
+     * @return string string a URL-encoded string.
+     */
+    function httpBuildQuery($data, $parentKey = '', $isNested = false)
```

Comment:
Ahh OK, fair enough! Would you like me to update the PR with the one regex version?